scripts/deploy/09_eth_put_theta_vault.ts

```diff
@@ -0,0 +1,163 @@
+import { run } from "hardhat";
+import { HardhatRuntimeEnvironment } from "hardhat/types";
+import {
+  CHAINID,
+  WETH_ADDRESS,
+  ETH_USDC_POOL,
+  USDC_ADDRESS,
+  USDC_PRICE_ORACLE,
+  ETH_PRICE_ORACLE,
+  OptionsPremiumPricerInStables_BYTECODE,
+} from "../../constants/constants";
+import OptionsPremiumPricerInStables_ABI from "../../constants/abis/OptionsPremiumPricerInStables.json";
+import {
+  AUCTION_DURATION,
+  AVAX_STRIKE_STEP,
+  ETH_STRIKE_STEP,
+  MANAGEMENT_FEE,
+  PERFORMANCE_FEE,
+  PREMIUM_DISCOUNT,
+  STRIKE_DELTA,
+} from "../utils/constants";
+
+const TOKEN_NAME = {
+  [CHAINID.ETH_MAINNET]: 'Ribbon USDC Theta Vault ETH Put',
+  [CHAINID.ETH_KOVAN]: 'Ribbon USDC Theta Vault ETH Put',
+  [CHAINID.AVAX_MAINNET]: 'Ribbon USDC Theta Vault AVAX Put',
+  [CHAINID.AVAX_FUJI]: 'Ribbon USDC Theta Vault AVAX Put',
+};
```

Comment:
Can safely [remove the name switch](https://github.com/ribbon-finance/ribbon-v2/pull/198/files#r774311005)